src/app/test_executive/zkapps.ml

```diff
@@ -406,6 +406,11 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       in
       [%log info] "ZkApp transactions included in transition frontier"
     in
+    let snark_work_event_subscription =
+      Event_router.on (event_router t) Snark_work_gossip ~f:(fun _ _ ->
```

Comment:
Can we instead add this to the network state? There we can count the number of snark work gossiped during the test, and still log this message out as a spam log. One thing to think of here though is that we will be receiving this event for each node that receives the gossip, I believe, so we may need to do some work to dedup those messages across nodes (or track which work was seen by which nodes so that we can check for consistency later).